bin/release_tag.ts

```diff
@@ -92,31 +92,19 @@ logger.info(`Fetching base "${origin}" ...`);
 exec(`git fetch ${origin}`);
 
 /**
- * @param {number} index - The tag name index
- * @returns {string} The new tag name
+ * @param index The tag name index
+ * @returns The new tag name
```

Comment:
You can completely remove these comments as they were only necessary to have valid JSDoc. :slightly_smiling_face: